index.d.ts

```diff
@@ -0,0 +1,4 @@
+/**
+Strip [ANSI escape codes](https://en.wikipedia.org/wiki/ANSI_escape_code).
+*/
```

Comment:
No need for this comment. Just adds more bytes to download.

Reply:
@sindresorhus usually wants doc comments in type definition files.

Reply:
Huh ok.

Reply:
See: https://github.com/sindresorhus/typescript-definition-style-guide

Reply:
Okay then keep it, makes sense actually since IDE's pick it up. Forget I said anything 😅